scripts/check_cdn_updates.py

```
@@ -0,0 +1,916 @@
+
+    return None
+
+
+def extract_version_from_url(url: str, package_name: str) -> Optional[str]:
+    """Extract version number from CDN URL"""
+    patterns = [
+        rf"{package_name}@(\d+\.\d+\.\d+)",
+        rf"{package_name}[/-](\d+\.\d+\.\d+)",
+        rf"{package_name}\.js[/@](\d+\.\d+\.\d+)",
+        rf"bootstrap(?:cdn)?/(?:bootstrap/)?(\d+\.\d+\.\d+)",  # Bootstrap specific
+        rf"datatables\.net[@/](\d+\.\d+\.\d+)",  # DataTables specific (@ or /)
```

Ruff (0.15.21)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)